frontend/js/helpers/metric-boxes.js

```diff
@@ -1,180 +1,104 @@
-                            </div>
-                        </div>
-                        <div class="ui bottom right attached label icon" data-position="bottom right" data-inverted="" data-tooltip="Estimated external energy cost for network infrastructure. See details under formula.">
-                            <u><a href="https://www.green-coding.io/co2-formulas/">via Formula</a></u>
-                            <i class="question circle icon"></i>
-                        </div>
-                        <div class="ui bottom left attached label">
-                            <span class="metric-type"></span>
-                        </div>
-                    </div>
-                </div>
-            </div>
-            <div class="ui card machine-co2">
-                <div class="ui content">
-                    <div class="ui top black attached label overflow-ellipsis">Machine CO<sub>2</sub> (usage) <span class="si-unit"></span></div>
-                    <div class="description">
-                        <div class="ui fluid mini statistic">
-                            <div class="value">
-                                <i class="burn icon"></i> <span>N/A</span>
-                            </div>
-                        </div>
-                        <div class="ui bottom right attached label icon" data-position="bottom right" data-inverted="" data-tooltip="CO2 cost of usage phase">
-                            <u><a href="https://www.green-coding.io/co2-formulas/">via Formula</a></u>
-                            <i class="question circle icon"></i>
-                        </div>
-                        <div class="ui bottom left attached label">
-                            <span class="metric-type"></span>
-                        </div>
+        const hardwareCards = [
+                { key: 'cpu', name: 'CPU', icon: 'microchip', colour: 'orange', variable:true },
+                { key: 'gpu', name: 'GPU', icon: 'camera retro', colour: 'orange', variable:true  },
+                { key: 'dram', name: 'DRAM', icon: 'memory', colour: 'orange', variable:true },
+                { key: 'disk', name: 'Disk', icon: 'hdd', colour: 'orange', variable:true  },
+                { key: 'machine', name: 'Machine', icon: 'server', colour: 'orange', variable:true },
+        ];
+
+        const extraCards = [
+            { key: 'runtime', name: 'Phase Duration', icon: 'hourglass half', colour: 'green', variable:false },
+            { key: 'network-data', name: 'Data Transferred', icon: 'exchange', colour: 'green', variable:false },
+            { key: 'network', name: 'Network', icon: 'sitemap', colour: 'green', variable:true },
+            { key: 'embodied-carbon', name: 'Embodied Carbon', icon: 'industry', colour: 'green', variable:false },
+            { key: 'operational-carbon', name: 'Operational Carbon', icon: 'fire', colour: 'green', variable:false },
```

Comment:
(This location of the comment is not really accurate)
THe machine-co2 card does not switch to power. So atm there is no possiblity for me to see the total machine energy.

Since I deem this metric the most important I would argue and it is a direct controllable one I would argue for moving it to the "hardware" row and remove it from the impact row also.

Reply:
FYI: This prevented me from fixing the test. Will gladly do so after